centreon/packages/ui-context/src/defaults.ts

```diff
@@ -1,7 +1,7 @@
 import { User, ThemeMode, ListingVariant } from './types';
 
 const defaultUser: User = {
-  alias: '',
+  alias: 'toto',
```

Comment:
since it is not a test file, is it expected ?

Reply:
Yeah I'll remove it